### PR TITLE
Fix README.md

### PR DESCRIPTION
**Pull request by nkwate-kazim**
Make README.md readable following the Github Markdown guide.

### CODE REVIEW

**Comment by nkwate-kazim** [comment]
@dashohoxha Please review

File: README.md [hunk 1]
```diff
@@ -1,17 +1,40 @@
...
-    ds config 
-    Add to wsproxy: ds wsproxy add mailpile.example.org
-    Navigate to mailpile.example.org:33411
+   1. Installation First install ds: https://github.com/docker-scripts/ds#installation
+   
+   2. Clone the scripts from github: git clone https://github.com/.../mailpile.git
+   
+   3. Move the script into /opt/docker-scripts/: 
+   ```
+      cp mailpile /opt/docker-scripts/
```
Comment (dashohoxha) [inline README.md]: If there is a single command, you can also do it with single backtics, like this: \`cp mailpile /opt/docker-scripts/`

File: README.md [hunk 2]
```diff
@@ -1,17 +1,40 @@
...
+   ```
+   4. cd /opt/docker-scripts/
+   
+   5. Initialize a container directory for mailpile
+   ```
+      ds init mailpile @mailpile
+   ```
+   6. Initialize and fix the settings:
+   ```
```
Comment (dashohoxha) [inline README.md]: I think that it looks better if you indent these 3 backticks with white spaces.

File: README.md [hunk 3]
```diff
@@ -1,17 +1,40 @@
...
+   
+   7. Build image, create the container and configure it:
+      ```  
+      ds build
+      ds create
+      ds config 
+      ```
+   
+   9. Add to wsproxy: ds wsproxy add mailpile.example.org
```
Comment (dashohoxha) [inline README.md]: You can use single backticks here too.

File: README.md [hunk 4]
```diff
@@ -1,17 +1,40 @@
...
+   
+   11. Navigate to mailpile.example.org:33411
```
Comment (dashohoxha) [inline README.md]: This is not related to the format:
If you use `wsproxy`, then you should not need to specify that port (`:33411`) after the domain. This means that the configuration of your container needs some fixing.

**Comment by dashohoxha** [comment]
1. You will not be able to do the containerization with docker-scripts properly, unless you are familiar with it. So, I suggest you to follow these steps first: https://github.com/docker-scripts/freedombox/issues/1

2. You will not be able to containerize an application (for example `mailpile`) properly, unless you know it in details (for example how to install, configure, update, upgrade, backup, restore, etc.). So, besides studying the docs of the application, it may be useful to play with it (for example on a virtual machine).

This comment is actually valid for everyone (@docker-scripts/ds) not only for @nkwate-kazim .

**Comment by nkwate-kazim** [comment]
Thanks for the review. Let me make the changes

**Review by dashohoxha — APPROVED** [review]
It seems Ok and I am accepting it, but you can improve it later if needed.
